examples/experimental/i3s-17-and-debug/app-debug.js

```diff
@@ -342,6 +344,14 @@ export default class App extends PureComponent {
     }
   }
 
+  getBoundingVolumeColor(tile) {
+    const {boundingVolumeColorMode} = this.state.debugOptions;
+    const color =
+      this._colorsMap.getColor(tile, {coloredBy: boundingVolumeColorMode}) || DEFAULT_COLOR;
+
+    return [...color, DEFAULT_BG_OPACITY];
```

Comment:
A bit risky if the color already has an opacity, you could get length 5?

Adding types and JSDoc could perhaps help a little.

Reply:
Will Ad JSDoc in further PR. `getColor` method returns RGB format always.